runtime/ddr/module.xml

```diff
@@ -406,11 +416,13 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 				<include-if condition="spec.linux_390.*"/>
 				<include-if condition="spec.linux_ppc.*_gcc"/>
 				<include-if condition="spec.linux_x86.*"/>
+				<include-if condition="spec.linux_riscv64.*"/>
```

Comment:
Please put this `<include-if>` element (and the one added below) in sorted order.

Reply:
Updated as suggested above.